gh-data.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+
+# Based on https://github.com/WebKit/standards-positions/blob/main/summary.py
+
+import argparse, json, os, requests, re
+
+
+# Utilities
+def write_json(filename, data):
+    with open(filename, "w") as f:
+        json.dump(data, f, indent=2, separators=(",", ": "))
+        f.write("\n")
+
+
+# General processing
+def process(issues):
+    summary = []
+    for issue in issues:
+        if is_ignorable_issue(issue):
+            continue
+
+        summary_item = {"issue": int(issue["html_url"][issue["html_url"].rfind("/") + 1 :])}
+        summary_item.update(process_labels(issue["labels"]))
+        summary_item.update(process_body(issue))
+        if summary_item["title"] is None:
+            summary_item["title"] = re.sub(r"Request for [Pp]osition: ", "", issue["title"])
+
+        summary.append(summary_item)
+    write_json("gh-data-summary.json", summary)
+
+
+def is_ignorable_issue(issue):
+    if "pull_request" in issue:
+        return True
+    for label in issue["labels"]:
+        if label["name"] in ("duplicate", "invalid", "tooling", "proposal withdrawn"):
+            return True
+    return False
+
+
+def process_labels(labels):
+    position = None
+    venues = []
+    concerns = []
+    topics = []
+
+    for label in labels:
+        # Position
+        if label["name"] == "blocked":
+            assert position is None
+            position = "blocked"
+        elif label["name"].startswith("position: "):
+            assert position is None
+            position = label["name"][len("position: ") :]
```

Comment:
```suggestion
            position = label["name"].removeprefix("position: ")
```

&c...

Though maybe `label["name"].split(": ")` is a better implement.